<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sketch Based Shape Retrieval</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpenOFF"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Tool</string>
    </property>
    <addaction name="actionGALIFTest"/>
    <addaction name="actionGaborFilterTest"/>
    <addaction name="separator"/>
    <addaction name="actionParameterOptimization"/>
    <addaction name="separator"/>
    <addaction name="actionCollectOFFFiles"/>
   </widget>
   <widget class="QMenu" name="menuRendering">
    <property name="title">
     <string>Rendering</string>
    </property>
    <addaction name="actionRenderingRegular"/>
    <addaction name="actionRenderingLine"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTool"/>
   <addaction name="menuRendering"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionGreedyInverse">
   <property name="text">
    <string>&amp;Greedy Inverse</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionSaveImage">
   <property name="text">
    <string>Save Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionModeSketch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sketch</string>
   </property>
  </action>
  <action name="actionMode3DView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3D View</string>
   </property>
  </action>
  <action name="actionRandomGeneration_2">
   <property name="text">
    <string>Random Generation</string>
   </property>
  </action>
  <action name="actionRandomGeneration">
   <property name="text">
    <string>&amp;Random Generation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionLoadSketch">
   <property name="text">
    <string>&amp;Load Sketch</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveSketch">
   <property name="text">
    <string>&amp;Save Sketch</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionNewSketch">
   <property name="text">
    <string>&amp;New Sketch</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionPenColorBranch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Branch</string>
   </property>
  </action>
  <action name="actionPenColorLeaf">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Leaf</string>
   </property>
  </action>
  <action name="actionPenWidth20">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>20</string>
   </property>
  </action>
  <action name="actionPenWidth10">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </action>
  <action name="actionPenWidth5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </action>
  <action name="actionLoadCamera">
   <property name="text">
    <string>Load Camera</string>
   </property>
  </action>
  <action name="actionSaveCamera">
   <property name="text">
    <string>Save Camera</string>
   </property>
  </action>
  <action name="actionResetCamera">
   <property name="text">
    <string>Reset Camera</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionStepFloor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Floor</string>
   </property>
  </action>
  <action name="actionStepWindow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Window</string>
   </property>
  </action>
  <action name="actionViewWireFrame">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>WireFrame</string>
   </property>
   <property name="shortcut">
    <string>W</string>
   </property>
  </action>
  <action name="actionAnalyzeRules">
   <property name="text">
    <string>Analyze Rules</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionFindMatchingRule">
   <property name="text">
    <string>Find Matching Rule</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionClear3DModel">
   <property name="text">
    <string>Clear 3D Model</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionOpenOFF">
   <property name="text">
    <string>Open OFF</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionGALIFTest">
   <property name="text">
    <string>GALIF Test</string>
   </property>
  </action>
  <action name="actionDFT">
   <property name="text">
    <string>DFT</string>
   </property>
  </action>
  <action name="actionGabor_Filter">
   <property name="text">
    <string>Gabor Filter</string>
   </property>
  </action>
  <action name="actionIDFT">
   <property name="text">
    <string>IDFT</string>
   </property>
  </action>
  <action name="actionCollectOFFFiles">
   <property name="text">
    <string>Collect OFF Files</string>
   </property>
  </action>
  <action name="actionGaborFilterTest">
   <property name="text">
    <string>Gabor Filter Test</string>
   </property>
  </action>
  <action name="actionRenderingRegular">
   <property name="text">
    <string>Regular</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="actionRenderingLine">
   <property name="text">
    <string>Line</string>
   </property>
   <property name="shortcut">
    <string>L</string>
   </property>
  </action>
  <action name="actionParameterOptimization">
   <property name="text">
    <string>Parameter Optimization</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
